### ISSUE TITLE
Can't download subtitle if "Encode subtitles to UTF8" is unchecked

### ISSUE THREAD

**Issue by netcop82** [morpheus65535/bazarr#548]
Hi,

This possible bug is somehow related with the one described here: https://github.com/morpheus65535/bazarr/issues/38

**If "Encode subtitles to UTF8" is checked**, all the subtitles (pt-BR) I download get weird characters, like the ones described in the issue referenced above.

**If "Encode subtitles to UTF8" is unchecked**, some (but not all) subtitles don't get downloaded, and bazarr throws the following error:

> Message:
> BAZARR Error saving subtitles file to disk for this file:/movies/Movie/movie.mkv
> 
> Exception:
> Traceback (most recent call last):
>  File "/app/bazarr/bazarr/get_subtitle.py", line 199, in download_subtitle
>   path_decoder=force_unicode
>  File "/app/bazarr/bazarr/../libs/subliminal_patch/core.py", line 853, in save_subtitles
>   content = subtitle.get_modified_content(format=format, debug=debug_mods)
>  File "/app/bazarr/bazarr/../libs/subliminal_patch/subtitle.py", line 323, in get_modified_content
>   encoding=self._guessed_encoding)
> UnicodeEncodeError: \latin-1\ codec can\t encode character u\\\u201c\ in position 153: ordinal not in range(256)

I have Bazarr in a docker (linuxserver arm compatible image), version 0.8.1, on a Raspberry Pi 3 b+.

Thanks in advance for any clarification.

**Comment by morpheus65535** [comment]
What is your default encoding? `locale`

Maybe take a look at this: https://github.com/morpheus65535/bazarr/issues/421

**Comment by netcop82** [comment]
> What is your default encoding? `locale`
> 
> Maybe take a look at this: #421

Thanks for the reply!

This is the result:

_root@raspberrypi:~# locale
LANG=en_US.UTF-8
LANGUAGE=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES=en_US.UTF-8
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=_

I tried the solution _nmolitor87_ provided for the issue #421, but my locale results were already "en_US.UTF-8", so nothing changed.

Should I try to change my locale to "pt_BR.UTF-8" or "pt_BR ISO-8859-1"?

**Comment by morpheus65535** [comment]
LC_ALL is empty and should not.

**Comment by netcop82** [comment]
OK, I added a ``` LC_ALL="en_US.utf8" ``` to /etc/environment , and reboot the machine. Now when i type ``` locale ``` it look like this:

```
root@raspberrypi:~# locale
LANG=en_US.UTF-8
LANGUAGE=en_US.UTF-8
LC_CTYPE="en_US.utf8"
LC_NUMERIC="en_US.utf8"
LC_TIME="en_US.utf8"
LC_COLLATE="en_US.utf8"
LC_MONETARY="en_US.utf8"
LC_MESSAGES="en_US.utf8"
LC_PAPER="en_US.utf8"
LC_NAME="en_US.utf8"
LC_ADDRESS="en_US.utf8"
LC_TELEPHONE="en_US.utf8"
LC_MEASUREMENT="en_US.utf8"
LC_IDENTIFICATION="en_US.utf8"
LC_ALL=en_US.utf8
```
The issue still remains, though.

**Comment by morpheus65535** [comment]
Did you have enabled `Encode subtitles to UTF8` before testing?

What is returned by those? 
`python --version`
`python -c "import sys; print sys.getdefaultencoding();"`

**Comment by netcop82** [comment]
> What is returned by those?
> ``` python --version 1```
> ``` python -c "import sys; print sys.getdefaultencoding();" ```

```
root@raspberrypi:~# python --version
Python 2.7.13
root@raspberrypi:~# python -c "import sys; print sys.getdefaultencoding();"
ascii
root@raspberrypi:~#
```

But hey! I think I found out what was happening, and this is really dumb... The machine I'm messing around is a remote machine, and I didn't test the subtitles on Plex or anything. I was getting that weird characters from viewing the subtitles on the WinSCP internal editor, and, as I just found out, the internal editor was configured to display ANSI. When I changed that to UT-8, voilá, the UTF-8 converted subtitles are displaying characters correctly.

Now i cant just let "Encode subtitles to UT8" checked, and it is all good!

So sorry for my noobishness... thank you very much for helping me anyway!
